src/generateNumberCards.js

```diff
@@ -0,0 +1,25 @@
+function generateRandomInt(max, min) {
+    return Math.floor(Math.random() * (max - min + 1) + min)
+}
+
+function generate3To5NumberCards(){
+let randomNumbersHolder = {}
+    for (let i = 0; i < generateRandomInt(5,3); i++) {
+        let randomNumber = generateRandomInt(100, 0)
+        while (Object.values(randomNumbersHolder).includes(randomNumber)) {
+            randomNumber = generateRandomInt(100, 0)
+        } 
+        randomNumbersHolder[i] = randomNumber
+    }
+
+Object.values(randomNumbersHolder).forEach(number => {
+    let content = document.querySelector('.number-cards-container').innerHTML
+    content +=  
+        `<div class="number-card">
+            <p>${number}</p>
+        </div>`
+    document.querySelector('.number-cards-container').innerHTML = content
```

Comment:
why are you putting this into a variable, changing the variable then reassigning to the dom? just assign straight to the dom:
```
document.querySelector('.number-cards-container').innerHTML +=         `<div class="number-card">
            <p>${number}</p>
        </div>`
```